blog/docs/articles/How-IBM-watsonx-Assistant-uses-Knative-Eventing-to-train-machine-learning-models.md

```diff
@@ -0,0 +1,9 @@
+# How IBM Watsonx Assistant uses Knative Eventing to train machine learning models
+
+![How IBM Watsonx Assistant uses Knative Eventing to train machine learning models](./images/How-IBM-watsonx-Assistant-uses-Knative-Eventing-to-train-machine-learning-models.png)
+
+Read the full case study on the CNCF website: https://www.cncf.io/case-studies/ibmwatsonxassistant/
+
+Thanks to [Firat Bezir](https://www.linkedin.com/in/firatbezir/)
+and [Zainab Husain](https://www.linkedin.com/in/zainabhusain1/) for creating the impact visual
+summary.
```

Comment:
@firatbezir and @zainabhusain227 can you confirm if these links are correct?

Reply:
it is correct :)